code/modules/atmospherics/machinery/portable/pump.dm

```diff
@@ -107,8 +105,10 @@
 /obj/machinery/portable_atmospherics/pump/ui_data()
 	var/data = list()
 	data["on"] = on
-	data["direction"] = direction == PUMP_IN ? TRUE : FALSE
+	data["into_pump_or_port"] = into_pump_or_port ? TRUE : FALSE
 	data["connected"] = connected_port ? TRUE : FALSE
+	data["pump_or_port"] = connected_port ? "Port" : "Pump"
```

Comment:
This data can (and should) be easily derived in JS. You might need to expose the "holding" var to JS.

Also while at it, please get rid of every `? TRUE : FALSE`, because it's painful to look at. Thank you.